fire/fire_test.py

```diff
@@ -288,12 +288,12 @@ def fn3(arg, **kwargs):
                      ('value', {'nothing': False}))
 
   def testTraceFlag(self):
-    self.assertIsInstance(
-        fire.Fire(tc.BoolConverter, 'as-bool True -- --trace'), trace.FireTrace)
-    self.assertIsInstance(
-        fire.Fire(tc.BoolConverter, 'as-bool True -- -t'), trace.FireTrace)
-    self.assertIsInstance(
-        fire.Fire(tc.BoolConverter, '-- --trace'), trace.FireTrace)
+    with self.assertRaisesFireExit(0, 'Fire trace:\n'):
+      fire.Fire(tc.BoolConverter, 'as-bool True -- --trace')
```

Comment:
only issue with switching to raising FireExit(0) on trace arg is it makes it a bit harder to work with interactively (e.g., can accidentally cause your shell to exit if you don't catch the exception) - but I guess you'd use the `--interactive` command line arg to do that instead?